### PR TITLE
Add recipe for SpotHit Notifier bridge

### PR DESCRIPTION
**Pull request by JamesHemery**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | symfony/symfony-docs#14869

### CODE REVIEW

File: symfony/spot-hit-notifier/5.3/manifest.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "#1": "SPOTHIT_DSN=spothit://api_token@default?from=MyCompany"
```
Comment (OskarStark) [inline symfony/spot-hit-notifier/5.3/manifest.json:3]:
```suggestion
        "#1": "SPOTHIT_DSN=spothit://TOKEN@default?from=FROM"
```

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Review by OskarStark — APPROVED** [review]

File: symfony/spot-hit-notifier/5.3/manifest.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (OskarStark) [inline symfony/spot-hit-notifier/5.3/manifest.json:1]: Please rename the folder to `spothit-notifier`

**Review by OskarStark — APPROVED** [review]

**Comment by fabpot** [comment]
@symfony-flex-server review again please

**Comment by fabpot** [comment]
@symfony-flex-server review again please

**Comment by fabpot** [comment]
@symfony-flex-server review again please

**Comment by fabpot** [comment]
Can you rename the directory to `spot-hit-notifier`?

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Pull request merged**